_timeline/2018-07-13-state-of-our-networks-2018.md

```diff
@@ -2,4 +2,4 @@
 title: State of Our Networks 2018
 date: 2018-07-13
 ---
-Members helped organize the second [State of Our Networks](https://ournetworks.ca/2018) in Toronto, with the theme of _Beyond DIY: Do It With Others_. Toronto Mesh members [streamed the conference over IPFS](https://github.com/tomeshnet/ipfs-live-streaming). Recorded videos will be published shortly.
+Members helped organize the second [State of Our Networks](https://ournetworks.ca/2018) in Toronto, with the theme of _Beyond DIY: Do It With Others_. Toronto Mesh members [streamed the conference over IPFS](https://github.com/tomeshnet/ipfs-live-streaming). [Recorded videos](https://ournetworks.ca/conference/recorded-talks/) of the talks archived on [IPFS](https://ipfs.io/ipfs/QmQQfD1ztg1aG82VmUmutusbxn7tL4c2o1qHv1ivyWbNj6/) and [YouTube](https://www.youtube.com/playlist?list=PLx7_J32Ys60ey2bgSn2soAoBy0v9bBUbT).
```

Comment:
The permanent link that we should use is https://ournetworks.ca/2018... that will always redirect to a snapshot of the 2018 version of the site. I'm not sure about including the Youtube/IPFS link, all your call

Reply:
https://ournetworks.ca/2018/conference/recorded-talks/ does not work. Was hoping to point it directly to the talks page.

Reply:
I'm saying that https://ournetworks.ca/2018 is gonna persist as a way to get folks to the 2018 content via redirect, there is a banner that will link to the recorded talks (we are currently thinking subdomains so 2018.ournetworks.ca/....) but I do not know exactly what the future ia will be for talks on the main page once that gets updated 🤷‍♀️

Reply:
@darkdrgn2k I changed link to main page as @dcwalk pointed out, because when 2019 comes along the deep link will be broken